packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.spec.ts

```diff
@@ -94,7 +76,7 @@ describe('PreferencesRegistriesEditing', () => {
     render(PreferencesRegistriesEditing);
     const addRegistryBtn = screen.getByRole('button', { name: 'Add registry' });
     await userEvent.click(addRegistryBtn);
-    const button = screen.getByRole('button', { name: 'Add' });
+    let button = screen.getByRole('button', { name: 'Add' });
```

Comment:
nitpick: we could use a helper function to avoid repetition, but this is just a nitpick (not blocking)

```suggestion
    const getButton = (): HTMLElement => screen.getByRole('button', { name: 'Add' });
    let button = getButton();
```